spec/performance/responses_test.rb

```diff
@@ -0,0 +1,32 @@
+context 'performance' do
```

Comment:
Je hebt wel een punt. Met een rake task kun je de volledige tijd benchmarken die je nodig hebt om een invulling te doen, inclusief de tijd dat je wacht op de R backend.
Deze benchmark spec gebruikt VCR en haalt dus de tijd die R nodig heeft uit de vergelijking, zo meet je alleen de snelheid van de Ruby code.
Die rake task kunnen we zeker nog wel maken voor een 'totaal' benchmark.